libs/core/topology/src/topology.cpp

```diff
@@ -951,7 +951,7 @@ namespace hpx { namespace threads {
                 break;
 
             case HWLOC_OBJ_NODE:
-                print_info(os, obj, "Node ", comma);
+                print_info(os, obj, "NumaNode ", comma);
```

Comment:
Nitpick: `NUMANode`? Only because `NUMA` is an acronym and that's the capitalization hwloc uses as well. Either is fine by me though.